src/main/java/dinkplugin/notifiers/LootNotifier.java

```diff
@@ -131,6 +131,11 @@ public void onLootReceived(LootReceived lootReceived) {
 
         // only consider non-NPC and non-PK loot
         if (lootReceived.getType() == LootRecordType.EVENT || lootReceived.getType() == LootRecordType.PICKPOCKET) {
+            if ("Barbarian Assault high gamble".equals(lootReceived.getName()) && !config.lootIncludeGambles()) {
```

Comment:
would an inverted check here be more efficient (i.e. check config first, then string)? it's equally readable imo

Reply:
this order is more efficient (the string length check on the loot source is fast while the config object has proxy reflection overhead)